content/docs/csidriver/features/powerflex.md

```diff
@@ -393,19 +393,28 @@ To manage multiple arrays you need to create an array connection configuration t
 
 ### Creating array configuration
 
-There is a sample yaml file in the samples folder under the top-level directory called `config.yaml` with the following content:
+There is a sample yaml file in the samples folder called `secret.yaml` with the following content:
  ```yaml
-  # Username for accessing PowerFlex system.	
-- username: "admin"
-  # Password for accessing PowerFlex system.	
+  # Username for accessing PowerFlex system.
+  # If authorization is enabled, username will be ignored.
+  - username: "admin"
+  # Password for accessing PowerFlex system.
+  # If authorization is enabled, password will be ignored.
   password: "password"
-  # System name/ID of PowerFlex system.	
-  systemID: "ID1"
+  # PowerFlex system name or ID.	
+  # Required: true
+  systemID: "1a99aa999999aa9a"
+  # Required: false
+  # Previous names used in secret of PowerFlex system. Only needed if PowerFlex System Name has been changed by user 
+  # and old resources are still based on the old name. 
+  allSystemNames: "pflex-1,pflex-2"
```

Comment:
Is it deprecated ? If so let's mark it in the comments

Reply:
allSystemNames is not deprecated, its use case might be limited, but we still use this in our nightly tests

Reply:
This isn't depreciated, rather it's a place to reference previous names used in older installs on the user's system.

Reply:
This brings an interesting point. If the system names have changed but the systemId remains constant, then maybe we can still be backward compatible, but then we might lose flexibility of using either system name or systemId and be reliant only on the ID.

Reply:
under the covers, we are reliant on ID only:
https://github.com/dell/csi-powerflex/blob/73b1e6874864e1a222999271e3e5988ef3433dd8/service/service.go#L754 
we have a map that goes from name (and any old name) -> ID, because ID will never change

Reply:
reason why we need this field is because for a slim amount of releases (I think it might have been just 1.5 or anything lower) 
we were not reliant on ID only. So csivolid could be systemName-pflexID in these older versions and if systemName changed, pflex driver would not be able to do any operations on volume since it wouldn't have the corresponding systemName

Reply:
> Makes me think if we should drop system name altogether :) that is one less field in the yaml and less manipulation on the back-end :)

I agree but I think a lot of customers prefer name over ID 😃

Reply:
I think we should just say "name or ID" everywhere 
if we drop one, customers that use it might ask

Reply:
and maybe change secret field from systemId -> system:

Reply:
Maybe we change the field to be system: (and in comments capture this could be user friendly system name or hexadecimal systemId). We can revisit this later as part of the global simplification in deployment.